packages/vuetify/src/components/VIcon/VIcon.ts

```diff
@@ -34,7 +34,12 @@ const VIcon = mixins(
     disabled: Boolean,
     left: Boolean,
     right: Boolean,
-    dense: Boolean
+    dense: Boolean,
+    tag: {
+      type: String,
+      required: false,
+      default: 'i'
+    },
```

Comment:
@FRFlor please remove trailing comma

Reply:
Sorry about that, the issue should be fixed now.